stats/cumulative_total_proj_summary_graph.inc

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
Please move this file to `pinc/graph_data.inc` or some other similarly-named, more-generic file we can use for more graph data functions.